quickwit/quickwit-cli/Cargo.toml

```diff
@@ -65,6 +65,7 @@ quickwit-doc-mapper = { workspace = true }
 quickwit-indexing = { workspace = true }
 quickwit-metastore = { workspace = true }
 quickwit-proto = { workspace = true }
+quickwit-query = { workspace = true }
```

Comment:
this is not looking good...

I understand you want to add this feature to forward the feature to quickwit-query

Reply:
Investigated a little bit more..

So originally we have 
a) `quickwit-query` <- `quickwit-doc-mapper`
b) `quickwit-query` <- `quickiwtindexing`

That are quite discutable.
b) exists to get the tokenizer managers for indexing, but also unfortunately, due the delete pipeline which gets a query. The latter will make it difficult to inverse.

a) is less justified. I have previously opened a ticket to refactor this dependency away.
I dont think your PR reinforce that link so I guess it should be fine.

Reply:
Marking as resolved to avoid confusing.